Cargo.toml

```diff
@@ -40,6 +40,8 @@ goblin = "0.2.0"
 tui = { version = "0.12.0", default-features = false, features = ["crossterm"] }
 anyhow = "1.0.31"
 textwrap = "0.12.1"
+defmt-decoder = { git = "https://github.com/knurling-rs/defmt", rev = "1bb6478", version = "0.1.0", features = ['unstable'] }
+defmt-elf2table = { git = "https://github.com/knurling-rs/defmt", rev = "1bb6478", version = "0.1.0", features = ['unstable'] }
```

Comment:
This can't be published to crates.io unfortunately. Can we use `defmt` 0.1.2 from crates.io instead?

Reply:
I want to believe that at the time I did that, decoder and elf2table were not published but I could be wrong and maybe just missed something. This should certainly best be used with the crates.io. versions when merging.